apps/api/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "@latitude-data/typescript-config/base.json",
   "compilerOptions": {
+    "moduleResolution": "Bundler",

```

Comment:
> 'bundler' for use with bundlers. Like node16 and nodenext, this mode supports package.json "imports" and "exports", but unlike the Node.js resolution modes, bundler never requires file extensions on relative paths in imports.

[Reference](https://www.typescriptlang.org/tsconfig/#moduleResolution)

I think `Bundler` is ok

Reply:
why do u needed this change

Reply:
For `t3-env/core`. It use a module system that requires the more modern version. But the thing is we're already using `Bundler` in NextJS app.

Reply:
An I think `bundler` makes a lot of sense for us because we're using one only final bundler in nextjs or api and not having by package bundlers